marshmallow/schema.py

```diff
@@ -452,7 +452,7 @@ def accessor(cls, func):
 
     ##### Serialization/Deserialization API #####
 
-    def dump(self, obj, many=None, update_fields=True, **kwargs):
+    def dump(self, obj, many=None, update_fields=True, only=None, **kwargs):
```

Comment:
Ok, that's a good point. I'll see, if I can implement my use cases without this. What's the rationale for specifying `many` here? There are three and a half places where the schema can be configured (Meta, __init__, and dump/load); I don't see the logic yet, what goes where and why.